hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieSnapshotExporter.java

```diff
@@ -113,21 +126,7 @@ public int export(SparkSession spark, Config cfg) throws IOException {
         return -1;
       }
       if (!cfg.outputFormat.equalsIgnoreCase("hudi")) {
```

Comment:
"hudi" would be constant?

Reply:
Sure. Will fix this once #1404 merged, with explicit format checking.